### ISSUE TITLE
Games fail to boot when using LayeredFS mods with many files

### ISSUE THREAD

**Issue by Morph1984** [yuzu-emu/yuzu#7308]
Some mods, such as [Zelda's Ballad & More](https://gamebanana.com/mods/246725) will cause the game to fail to boot due to too many open files.

Reported by d2hpdG1hbmNo#0195 on yuzu discord

[yuzu_log (2).txt](https://github.com/yuzu-emu/yuzu/files/7508895/yuzu_log.2.txt)

**Comment by Morph1984** [comment]
Test

**Comment by konakona0** [comment]
This issue occurred for me as well.
Recently tried to load Zelda's Ballad in Yuzu, built from source in Arch Linux 5.18.10. Yuzu will load shaders, attempt to load mods, then will print the following message to stdout before dumping core and exiting.
![image](https://user-images.githubusercontent.com/28904845/178732902-ae4275fc-06d6-419b-8afb-41031ad0f060.png)

This crash seems to happen even if the mods are unchecked, I tried running BotW with the mods disabled, the problem persisted until the files were removed from the `load` folder.

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9102055/yuzu_log.txt)
